bricks/vs/src/code-editor/index.css

```diff
@@ -1,5 +1,5 @@
 .highlight {
-  color: rgb(74 170 189) !important;
+  text-decoration-color: rgb(74 170 189);
   text-decoration: underline;
   font-weight: bold;
   font-style: oblique;
```

Comment:
这个地方我建议改下，只要下划线，加重和斜体没有太大必要。

另外下划线用 box-shadow 实现，这样下划线颜色就一致了，另外文字不会因为下划线贴的太近而影响阅读

```suggestion
  box-shadow: rgb(74 170 189) 0px 1px 0px 0px;
```

Reply:
看下对比：

<img width="429" alt="image" src="https://github.com/easyops-cn/next-bricks/assets/2338946/ca5d2391-3698-4933-82a9-58ec75fe77be">

<img width="430" alt="image" src="https://github.com/easyops-cn/next-bricks/assets/2338946/15fc52a6-37f1-4422-b2e3-f38326d2e902">